.changes/unreleased/Fixes-20221117-220320.yaml

```
@@ -0,0 +1,7 @@
+kind: Fixes
+body: Fix typo in util.py
+time: 2022-11-17T22:03:20.4836855+09:00
+custom:
+  Author: eltociear
+  Issue: "6037"
```

We use a specific issue for changes that do not have associated issues.  It's the result of how our changelog is currently implemented.
```suggestion
  Issue: "4904"
```